.github/pull_request_template.md

```diff
@@ -26,8 +26,13 @@
 
 ### Reviewers
 
-<!-- 
+<!--
     Optional
     Request additional reviewers with @username
 -->
 
+### Release Notes
+<!--
+    Optional
+    In case of breaking changes, functionality changes or refactors, plase add release notes here. For more information, please see [the contributing guidelines](https://os.mbed.com/docs/mbed-os/latest/contributing/workflow.html#pull-request-types).
```

Comment:
@adbridge Just to check to have this implemented - we documented in docs that any functional change, but rather as it's here that even bigger refactors or breaking change (it's functionality change but has it's own item) can have release notes, would make this requirement: script should check any PR for release notes and fetch them.

Reply:
> plase add

Should say 'please add'